browser.js

```diff
@@ -0,0 +1,23 @@
+const {Tracer, BatchRecorder, ExplicitContext} = require('zipkin');
+const {HttpLogger} = require('zipkin-transport-http');
+
+// Send spans to the origin server under the path /zipkin
+const recorder = new BatchRecorder({
+  logger: new HttpLogger({
+    endpoint: location.origin + '/zipkin'
```

Comment:
fyi @hyleung @shakuzen I toyed around with proxy mounting zipkin so there's no origin problems, nor configuration needed on the client browser side.